src/v2/Components/FlashBanner/__tests__/FlashBanner.jest.tsx

```diff
@@ -1,22 +1,62 @@
-import { mount } from "enzyme"
+import { ReactWrapper, mount } from "enzyme"
 import "jest-styled-components"
 import React from "react"
-import { FlashBanner } from "v2/Components/FlashBanner"
+import { BaseFlashBanner } from "v2/Components/FlashBanner"
+import { renderRelayTree } from "v2/DevTools"
+import { FlashBannerTestQueryRawResponse } from "v2/__generated__/FlashBannerTestQuery.graphql"
+import { graphql } from "react-relay"
+import { Banner } from "@artsy/palette"
+
+jest.unmock("react-tracking")
+jest.unmock("react-relay")
 
 const { location: originalLocation } = window
 
-describe("FlashBanner", () => {
-  afterEach(() => {
-    window.location = originalLocation
+const mockMe: FlashBannerTestQueryRawResponse["me"] = {
+  id: "fff",
+  canRequestEmailConfirmation: false,
+}
+
+let wrapper: ReactWrapper
+
+async function getWrapper(
+  mockData: FlashBannerTestQueryRawResponse,
+  queryString: string = ""
+) {
+  delete window.location
+  const fakeLocation: any = { search: queryString }
+  window.location = fakeLocation
+
+  wrapper = await renderRelayTree({
```

Comment:
Why initialize the variable outside the function?

Reply:
was trying to debug the testing issue i have commented out below so wanted to be able to call `wrapper.unmount()` in the teardown `afterEach()` callback

Reply:
Probably not necessary, we typically don't unmount or teardown `renderRelayTree` calls